server/app.js

```diff
@@ -1,13 +1,20 @@
 const path = require('path');
 const express = require('express');
 const morgan = require('morgan');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
```

Comment:
It is a middleware that runs webpack on your server, so that you don't have to do a webpack or webpack -w to recreate your bundle.js from the terminal.  The web server will automatically do it for you.  It was listed as a required tech on learn.makerpass.com so I included it